src/helper.ts

```diff
@@ -105,24 +105,16 @@ const validateInput = (
   return { isValid, errorMessage };
 };
 
-const filterObject = (queryObject: QueryObject): QueryObject => {
+const filterObject = (
```

Comment:
are you sure this function returns the required result. You are literally equating rules to the required fields list? Am assuming the list is something like `['id', 'errorMessage']`, so saying rules = requiredFields, seems off. I would expect that you apply a filter to the queryObject rules field, and only return the fields that match what is in the requiredFields array

Reply:
The list is an array of objects `[{id: '1', errorMessage:''}]`

Reply:
Hmmm, it shouldn't be that way... If you are choosing to filter.. The purpose of the function loses meaning